api/src/main/java/org/eclipse/microprofile/faulttolerance/Retry.java

```diff
@@ -109,4 +110,27 @@
     @Nonbinding
     Class<? extends Throwable>[] abortOn() default {};
 
+
+
+    /**
+     *
+     * When this value is greater than 1, backoff retry is activated.
+     * {@link #delay()} between retries will be increased exponentially with this value until
+     * the delay reaches {@link #backoffMaxDelay()}
+     * Value must be equals greater than 1
+     *
+     * @return the backoff factor
+     */
+    @Nonbinding
+    double backoffDelayFactor() default 1;
+
+
+    /**
+     *
+     * @return the maximum delay value. No limit if it's set to 0. Value is ignored if {@link #backoffDelayFactor()} is 1
+     */
+    @Nonbinding
+    long backoffMaxDelay() default 0;
```

Comment:
maxDelay must be great than delay. Otherwise, IllegalArgumentException should be thrown.

Reply:
how would you you deactivate backoff retry then?

Reply:
As discussed on the previous call, this method is not needed. We have delay specified already. @antoinesd will you be able to update this? If not, I can submit a PR.